pkg/bpf/bpf.go

```diff
@@ -121,10 +121,10 @@ func CreateMap(mapType int, keySize, valueSize, maxEntries, flags uint32) (int,
 		unsafe.Sizeof(uba),
 	)
 
-	if ret != 0 {
-		return int(ret), nil
+	if err != 0 {
+		return 0, fmt.Errorf("Unable to create map: %s", err)
```

Comment:
@joestringer this should be `Unable to create map: %d` and not `%s`

Reply:
Huh? `%s` turns this into a readable string like "Bad file descriptor".